README.md

```diff
@@ -44,7 +45,16 @@ For example:
 }
 ```
 
+### Options of module
+- `languages` : An list available languages
+- `defaultLanguage` : The default language ***⚠ Required: The value need defined in `languages`***
```

Comment:
How about making it optional? We could just use the first language as the default. That way we keep the configuration backwards compatible.